src/main/java/com/gtnewhorizons/gtnhgradle/PropertiesConfiguration.java

```diff
@@ -811,6 +965,12 @@ public void initFromProperties(Map<?, ?> props, BiConsumer<String, Object> onMis
             if (prop == null) {
                 continue;
             }
+            final String mcVersion = prop.mcVersion();
+            if (!mcVersion.isEmpty()) {
+                if (!GTNHGradlePlugin.MinecraftVersion.isAcceptableMinecraftVersion(mcVersion)) {
```

Comment:
This class is loaded in an isolated classloader during the update process, so it shouldn't use the GTNHGradlePlugin class at all or it will crash the script updater. Moving MinecraftVersion to not be an inner class and making sure it doesn't have any gradle or other library dependencies would be required to use it here